README.md

```diff
@@ -1 +1,9 @@
 # express-yourself
+
+assuming you have node installed, to install, you must:
+
+npm install (this will install dependencies. it will give you all the node modules you will need)
+
```

Comment:
it's generally considered best practice to put any code in back-ticks `, so the exact code is clear `npm install`